dell/precision/5490/default.nix

```
@@ -0,0 +1,15 @@
+{ lib, ... }:
+{
+  imports = [
+    ../../../common/gpu/nvidia
+  ];
+
+  # or even better: boot.kernelParams = pkgs.linuxPackages_latest;
+  boot.kernelParams = [ "i915.force_probe=7d55" ];
```

Reading the text above, sounds like we want more something like:

```suggestion
  boot.kernelParams = lib.mkIf (lib.versionOlder pkgs.linux.version "6.6")  [ "i915.force_probe=7d55" ];
  boot.kernelPackages = lib.mkIf (lib.versionOlder pkgs.linux.version "6.6") (lib.mkDefault pkgs.linuxPackages_latest);
```

Please replace 6.6 with the kernel version you think is the newest working one.